lms/djangoapps/mobile_api/users/views.py

```diff
@@ -107,15 +107,14 @@ def _last_visited_module_path(self, request, course):
             course.id, request.user, course, depth=2)
 
         course_module = get_module_for_descriptor(request.user, request, course, field_data_cache, course.id)
-        current = course_module
 
-        path = []
```

Comment:
Note to reviewers: These changes make this endpoint consistent with the behavior by the browser.